src/Tile_Maps.md

```diff
@@ -33,15 +33,53 @@ Bit 3    Tile VRAM Bank number      (0=Bank 0, 1=Bank 1)
 Bit 2-0  Background Palette number  (BGP0-7)
 ```
 
-When Bit 7 is set, the corresponding BG tile will have priority above
-all OBJs (regardless of the priority bits in OAM memory). There's also
-a Master Priority flag in LCDC register Bit 0 which overrides all other
-priority bits when cleared.
-
 Note that, if the map entry at `0:9800` is tile \$2A, the attribute at
 `1:9800` doesn't define properties for ALL tiles \$2A on-screen, but only
 the one at `0:9800`!
 
+### BG-to-OBJ Priority in CGB Mode
+
+In CGB Mode, the priority between the BG (and window) layer and the OBJ layer is declared in three different places:
+- [BG Map Attribute bit 7](<#BG Map Attributes (CGB Mode only)>)
+- [LCDC bit 0](<#LCDC.0 — BG and Window enable/priority>)
+- [OAM Attributes bit 7](<#[Byte 3 — Attributes/Flags](https://gbdev.io/pandocs/OAM.html#byte-3--attributesflags)>)
+
+We can infer the following rules from the table below:
+* If the BG color index is 0, the OBJ will always have priority;
+* Otherwise, if LCDC bit 0 is clear, the OBJ will always have priority;
+* Otherwise, if both the BG Attributes and the OAM Attributes bit 7 is clear, the OBJ will have priority;
+* Otherwise, BG will have priority.
+
+::: tip
+
+BG Map attributes bit 7 is used to override OAM attributes bit 7
```

Comment:
What is this saying?

Reply:
I'm not sure this is the best way to say it but I'm trying to explain the logical purpose of the BG Map attributes bit 7.
This is not a crucial detail but rather an interesting observation about the relation between those flags.

Reply:
I don't understand how it "overrides" the other bit, since their roles are symmetrical.

Reply:
I don't think their roles are symmetrical, in order to grant the BG layer priority over the OBJ layer you first need LCDC bit 0 to be set.
Only if the OBJ attributes bit 7 is clear you also need to set BG attributes bit 7 but if OBJ attributes bit 7 is set BG attributes bit 7 is not relevant and can be ignored (it's state doesn't matter and the BG will have priority).

This was rather not really intuitive to me at first so I thought mentioning this in a tip box could help the reader.

Reply:
I agree that LCDC bit 0 has a different role, but the role of BG and OBJ attributes' bit 7 is identical (besides the latter's inverted meaning).

Reply:
After more thinking I realized you are correct